source/declarations.tex

```diff
@@ -2364,8 +2364,11 @@
 \end{example}
 
 \pnum
-A type can also be named (often more easily) by using a
-\tcode{typedef}\iref{dcl.typedef}.
+\begin{note}
+Types can also be named with \grammarterm{typedef-name}{s},
+which are introduced by typedef declarations
+and \grammarterm{alias-declaration}{s}\iref{dcl.typedef}.
```

Comment:
Could you make the sentence singular throughout, please?

Reply:
Updated